preseed.cfg

```diff
@@ -2,11 +2,15 @@
 # kernel is to be installed.
 d-i base-installer/kernel/image string linux-image-novena
 
+# This command is run immediately before the partitioner starts. It may be
+# useful to apply dynamic partitioner preseeding that depends on the state
+# of the disks (which may not be visible when preseed/early_command runs).
+d-i preseed/early_command string mkdir /sd ; mount /dev/mmcblk0p1 /sd ; mv /sd/uInitrd /sd/uInitrd-install ; sync ; umount /sd ; sync ; rmdir /sd
```

Comment:
I did that assuming that novena-debian-support would eventually be updated to include the zzz-novena-mkimage. That way, the uInitrd install image would not be overriden when that happens.